tests/test.py

```diff
@@ -32,7 +32,7 @@ def test_metadata():
 
 def test_predict():
     model_endpoint = 'http://localhost:5000/model/predict'
-    file_path = 'assets/tom_cruise.jpg'
+    file_path = '../assets/tom_cruise.jpg'
```

Comment:
This caused Travis to fail

```suggestion
    file_path = 'assets/tom_cruise.jpg'
```

Reply:
```
>       with open(file_path, 'rb') as file:
E       FileNotFoundError: [Errno 2] No such file or directory: '../assets/tom_cruise.jpg'
```